aten/src/ATen/native/quantized/cpu/qnnpack/src/q8conv/8x8-aarch64-neon.S

```diff
@@ -102,9 +107,13 @@ BEGIN_FUNCTION pytorch_q8conv_ukernel_8x8__aarch64_neon
     # v23 := vacc7x4567
     MOV v23.16b, v9.16b
 
-    # Load multiplier
-    # - v26 = vmultiplier
-    LD1R {v26.4s}, [x8], 4
+    # Add offset to the base pointer
+    ADD x10, x10, x9
+    // Load requantization_scale
+    // - v26 = requantization_scale channels 0-3
+    // - v31 = requantization_scale channels 4-7
+    LD1 {v26.4s}, [x10], 16
+    LD1 {v30.4s}, [x10]
```

Comment:
This is a long shot but I am wondering if you can do this and remove `LSL x9, x9, 2` and `ADD x10, x10, x9` above, in case the value of x9 and x10 do not matter after this point:

```
LD1 {v26.4s}, [x10], x9, lsl 2
LD1 {v30.4s}, [x10, 16]
```

Reply:
Does that form exist?. I checked but it does not seem to me that the variant of LD1 exist where shift can be folded in.

Reply:
Yes so for LDR yes there is. Not for LD1. But I think this I can use in base + offset calculation done via add.